src/renderer/locales/de-DE.json

```diff
@@ -0,0 +1,107 @@
+  "settings.ambient_background.description": "Präsentiert eine Überlagerung des Coverbilds für das aktuell abgespielte Lied auf der Oberfläche",
+  "settings.ambient_background.spin_speed.title": "Drehgeschwindigkeit",
+  "settings.ambient_background.spin_speed.description": "Wie schnell sich das Hintergrundbild drehen soll",
+  "settings.ambient_background.opacity.title": "Transparenz",
+  "settings.ambient_background.opacity.description": "Wie sichtbar das Ambiente Bild über der App ist",
+  "settings.ambient_background.blur_strength.title": "Unschärfe-Stärke",
+  "settings.ambient_background.blur_strength.description": "Wie viel Unschärfe (in px) angewendet werden soll",
+  "settings.ambient_background.zoom.title": "Zoom",
+  "settings.ambient_background.zoom.description": "Passt die Skalierung des Ambienten Hintergrunds an, um sicherzustellen, dass das Cover den gesamten Bildschirm abdeckt",
+
+  "settings.playback_controls.title": "Wiedergabesteuerung",
+  "settings.playback_controls.description": "Wiedergabesteuerungen am unteren Rand von Amethyst anzeigen",
+
+  "settings.minimalist_mode.title": "Reduzierter Modus",
+  "settings.minimalist_mode.description": "Weniger auf der Benutzeroberfläche anzeigen, um sie sauber zu halten",
+
+  "settings.cover_art.title": "Coverbild",
+  "settings.cover_art.description": "Das Coverbild in der Nähe der Wiedergabesteuerung am unteren Rand der Oberfläche anzeigen",
+
+  "settings.debug_stats.title": "Debug-Statistiken",
+  "settings.debug_stats.description": "Zahlen wie Samples, FPS, Prozessorauslastung oben in der Aktionsleiste anzeigen",
+  
+  "settings.discord_rpc.description" : "Discord Rich Presence ermöglicht es dir, deine aktuelle Aktivität in Amethyst auf Discord zu teilen. Wenn aktiviert, aktualisiert die RPC-Funktion deinen Discord-Status, um die Version von Amethyst, das gerade abgespielte Lied oder die Playlist sowie Künstler und Album anzuzeigen.",
+  
+  "settings.auto_update.title": "Automatische Updates",
+  "settings.auto_update.description": "Automatisch nach Updates suchen und sie beim Start installieren",
+  
+  "settings.launch_on_startup.title": "Bei Systemstart ausführen",
+  "settings.launch_on_startup.description": "Amethyst beim Start des Systems starten",
+
+  "settings.language.title": "Sprache",
+  "settings.language.description": "Ändere die Sprache von Amethyst",
+
+  "settings.decibel_meter.title": "Dezibel-Messung",
+  "settings.decibel_meter.description": "Zeigt eine visuelle Darstellung des Schallpegels des Audiosignals in Dezibel (dB) an. Es zeigt den aktuellen Schallpegel des Audiosignals mit einem Diagramm an, bei dem die y-Achse den Schallpegel in Dezibel darstellt. Die Dezibel-Messung kann verwendet werden, um die Gesamtlautstärke des Audiosignals zu überwachen und sicherzustellen, dass sie innerhalb einer sicheren Hörgrenze liegt.",
+  "settings.decibel_meter.separate_pre_post.title": "Seperiere PRE und POST",
```

Comment:
I would Change "Seperiere PRE und POST" to "Trenne PRE und Post"

Reply:
Valid, could be a more user friendly term?
I'll update it in the next one